packages/rug/src/utils.js

```diff
@@ -26,10 +34,35 @@ export const calcModularScale = curry((scale, exponent) =>
   Math.round(base * Math.pow(scale, Math.round(exponent))) / base
 )
 
+/**
+ * Color Utilities
+ */
+
 export const addUnit = curry((unit, value) => `${value}${unit}`)
 
-export const opacify = (o, rgb) => rgb.replace('b(', 'ba(').replace(')', `,${o})`)
+export const opacify = (rgb, o = 1) => rgb.replace('b(', 'ba(').replace(')', `,${o})`)
```

Comment:
Shadows are a big part, but it's also one of the only performant ways to transition things, and in this context could also be used a lot.